akash1/HTML/ASSIn3/contact.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>contact</title>
+    <link rel="stylesheet" href="Assignment3Q2.html">
+    <link rel="stylesheet" href="MI9.html">
+
+<body>
```

Comment:
don't use body under head tag.

Reply:
i used for css

Reply:
dont have to use tag just u have to use body that is tag name